src/main/resources/interface-specification.yml

```diff
@@ -2,7 +2,7 @@ openapi: 3.0.3
 info:
   title: Interoperability API Gateway Micro Service
   description: exposes the API for interacting with interoperability features
-  version: '0.1.0'
+  version: '1.0'
```

Comment:
I guess this should be `{{version}}`, but let's keep it this way while we decide how to handle the `servers.url` properly 👍